search/api.py

```diff
@@ -111,17 +151,38 @@ def create_search_obj(user, search_param_dict=None, filter_on_email_optin=False)
     if search_param_dict is not None and search_param_dict.get('from') is not None:
         update_dict['from'] = search_param_dict['from']
     search_obj.update_from_dict(update_dict)
+
+    if not is_advance_search_capable:
+        # Derive visible fields from the ProfileLimitedSerializer
+        search_obj = search_obj.source(
+            include=[
+                'profile.*',
+                'program.total_courses',
```

Comment:
include parameter filters the values returned in the `_source` object, I think I added this code out of an abundance of paranoia, but since we have the separate doc type it is probably overkill now.